governance/MEMBERSHIP_SURVEY.md

```diff
@@ -0,0 +1,32 @@
+
+# Membership Survey
+
+Surveys for current and past members to collect data on member wants and needs. following #375 issue.
+
+## Suggested Questions
+
+@amiller-gh
```

Comment:
I would remove the credits for the questions, since they're questions from the CommComm as a whole. Generally crediting is almost always better, but presenting a unified front is probably best in this specific case :)